checks.d/mongo.py

```diff
@@ -922,11 +924,11 @@ def total_seconds(td):
                             # we tag the index
                             idx_tags = coll_tags + ["index:%s" % idx]
                             submit_method, metric_name_alias = \
-                                self._resolve_metric('collection.%s' % m, metrics_to_collect)
+                                self._resolve_metric('collection.%s' % m, self.COLLECTION_METRICS)
```

Comment:
It's always the same metric, but the tag is changing, isn't it ? Because you could put it outside of the `for` loop if it's the case 😛